tests/lib/config/config-file.js

```diff
@@ -349,6 +365,32 @@ describe("ConfigFile", function() {
             assert.deepEqual(config, updatedConfig.eslintConfig);
         });
 
+        it("should load fresh information from a .eslintrc.js file", function() {
+            var initialConfig = {
```

Comment:
I used the same technique as #4610 ([config-file.js#L317](https://github.com/eslint/eslint/blob/master/tests/lib/config/config-file.js#L317)). What would be the preferred way of doing it? Stubs?